ui/src/views/Settings.vue

```diff
@@ -110,25 +179,23 @@ export default {
     clearInterval(this.urlCheckInterval);
     next();
   },
-  created() {
-    this.getConfiguration();
-  },
   methods: {
     async getConfiguration() {
       this.loading.getConfiguration = true;
       this.error.getConfiguration = "";
       const taskAction = "get-configuration";
-      const eventId = this.getUuid();
 
       // register to task error
+      this.core.$root.$off(taskAction + "-aborted");
```

Comment:
Why was `eventId` removed? It is needed to handle the event properly.

Also, `this.core.$root.$off()` is not needed since we are using `$once` to register to the event.